diff --git a/dist/index.cjs b/dist/index.cjs
index 7333afd1d67937152030cca4c91eb220589b4ba1..5a7a26348bb8157dd7ae82789c15bb323ac87539 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -2563,7 +2563,6 @@ var getVitestSWCTransformConfig = ({
       type: isEsmProject && !useCjsModules ? "es6" : "commonjs"
     },
     disableNextSsg: !isPageFile,
-    disablePageConfig: true,
     isPageFile,
     pagesDir: nextDirectories.pagesDir,
     appDir: nextDirectories.appDir,
diff --git a/dist/index.js b/dist/index.js
index bdbb3bded64e7d7b0aefa76d1593b8c1ab0a6317..c404243bb778ffd21ac30c355f62a022c83ec337 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -2529,7 +2529,6 @@ var getVitestSWCTransformConfig = ({
       type: isEsmProject && !useCjsModules ? "es6" : "commonjs"
     },
     disableNextSsg: !isPageFile,
-    disablePageConfig: true,
     isPageFile,
     pagesDir: nextDirectories.pagesDir,
     appDir: nextDirectories.appDir,
